src/js/editor/editor.js

```diff
@@ -71,7 +71,8 @@ const defaults = {
   unknownCardHandler: () => {
     throw new Error('Unknown card encountered');
   },
-  mobiledoc: null
+  mobiledoc: null,
+  pasteInput: null
```

Comment:
`dom` felt pretty lame. Bleh, I guess it can be `html` and I can make it recognize when DOM is passed instead.